.github/workflows/ci.yml

```diff
@@ -13,5 +13,5 @@ on:
 
 jobs:
   call-ci:
-    uses: phoenix-rtos/phoenix-rtos-project/.github/workflows/ci-project.yml@master
+    uses: phoenix-rtos/phoenix-rtos-project/.github/workflows/ci-project.yml@nalajcie/build
```

Comment:
You might want to change this before merge, I guess

Reply:
Otherwise LGTM

Reply:
yeah, this is needed for every ci-related PR to actually test the changes - I'm always making it a separate commit to be deleted before the merge:
![image](https://github.com/phoenix-rtos/phoenix-rtos-project/assets/3939983/6e133af0-dfd4-4f0c-bd6a-f6161fae480d)